include/aunit/reporters/aunit-reporter-combine.ads

```diff
@@ -0,0 +1,26 @@
+with Ada.Containers.Vectors;
```

Comment:
This won't work for restricted runtimes, Ada.Containers are not present there

Reply:
Is there an easy way for me to test that? I tried adding `for Runtime ("Ada") use "zfp";` to the project using aunit, but then even AUnit.Time_Measure's usage of Ada.Calendar won't work.

Reply:
After reviewing aunit_shared.gpr it seems clear this currently works fine. 

I guess the proper way to do this would be to add another enumeration like the current Except, Calend, Memory and FileIO and then create a consistent interface to the Vectors package with two implementations, "vectors" would pass through to Ada.Containers.Vectors, while "novectors" would have some custom implementation.

Another option would be to use AUnit.Memory directly, or rather, to implement AUnit.Vectors in terms of AUnit.Memory, so that the above enumeration isn't necessay. Thoughts on which is better/simpler?

By the way, I saw that in aunit_shared.gpr around line 52 `when "cert" => ` the value for Calend remains "calendar", which is implicitly saying that Ada.Calendar is certified. It seems better to make that explicit if that is correct. If this were normal Ada code, I would say that these four enumerations are begging to be made into a record type, so that we could make the compiler spot such omissions, but I'm not sure that applies to gpr files.

Reply:
A third option I think is to move the trx reporter into a separate directory which is only included as a source when the runtime is set to full. This is the only option that doesn't have me unnecessarily supporting runtimes I don't use myself.